web/themes/custom/oc_bootstrap/css/style.css

```diff
@@ -990,3 +990,48 @@ article.teaser h2 a {
     width: 1000px;
   }
 }
+
+.division-building-materials-amp-construction,
+.division-consumer-electronics,
+.division-fashion,
+.division-fmcg-amp-cpg,
+.division-industrial-equipment-amp-packaging,
+.division-paper-publishing-amp-printing,
+.division-pharmaceutical,
+.division-transportation-amp-automotive {
+  min-height: 140px;
+  background-size: 100%;
+  background-position-y: center;
+  font-size: 20px;
+  font-weight: 700;
+  color: #FFFFFF;
+  text-transform: uppercase;
+  text-align: center;
+  vertical-align: middle;
+  margin: auto 0;
+}
+
+.division-building-materials-amp-construction {
```

Comment:
Once the division-row class is used, this selector changes to `.division-row.division-building-materials-amp-construction` to make sure it takes precedent